src/client/components/SchoolRegistration/StepOne/StepOne.jsx

```diff
@@ -0,0 +1,278 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import close from "../../../assets/images/close.png"
+import Logo from '../../../assets/images/katha-logo.png';
+import Button from '@material-ui/core/Button';
+import Typography from "@material-ui/core/Typography";
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import InputField from '../../common/TextField/InputField';
+import PaymentButton from '../../common/Button/PayButton';
+
+const useStyles = makeStyles(theme => ({
+    background: {
+        backgroundColor: '#FEDB50',
+        height: "500vh",
+        padding: 0,
+        width: "100%",
+        backgroundRepeat: "no-repeat",
+    },
+    RegistrationForm: {
+        color: "#66645E",
+        fontWeight: "400",
+        fontSize: "1rem"
+    },
+    registrationCard: {
+
+        [theme.breakpoints.down("xs")]: {
+            width: "85%"
+        },
+    },
+    registrationDivBackground: {
+        backgroundColor: '#FEDB50',
+        height: "25vh",
+        width: "100%",
+        backgroundSize: "cover",
+        backgroundPosition: "center",
+    },
+    UploadFile: {
+        width: 360,
+        height: 200,
+        textTransform: "none",
+        backgroundColor: "#FDF6D8",
+        "&:hover": {
+            backgroundColor: "#FDF6D8"
+        },
+        [theme.breakpoints.down("sm")]: {
+            width: 320
+        },
+        [theme.breakpoints.down("xs")]: {
+            width: 260
+        },
+
+    },
+    Reset: {
+        lineHeight: '24px',
+        fontWeight: 'bold',
+        fontSize: "0.65rem",
+        textTransform: "none",
+        [theme.breakpoints.down("sm")]: {
+            width: 320
+        },
+        [theme.breakpoints.down("xs")]: {
+            width: 260
+        },
+    },
+    supportedDocument: {
+        fontWeight: 400,
+        color: "#000000",
+        lineHeight: "1.25rem",
+    },
+    errorMessage: {
+        color: "#f44336",
+        fontWeight: 400,
+        lineHeight: "1rem",
+    }
+}));
+
+
+const StepOne = (props) => {
+    const classes = useStyles();
+
+
+    const [schoolName, setSchoolName] = useState('');
```

Comment:
Why these many states? Move it all into 1 state

Reply:
Since we handle validation in same page, to track  errors in run time and to track actual values we have 2 states for each field. Since we are passing setState function for another component we cannot add all values as single object in state.